<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDemo1Class</class>
 <widget class="QWidget" name="QtDemo1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtDemo1</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>1291</width>
     <height>711</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>981</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>矩形</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>旋转矩形</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>圆形</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>椭圆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>圆环</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>圆弧</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>多边形</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtDemo1.qrc"/>
 </resources>
 <connections/>
</ui>
